blockchain_geth/src/lib.rs

```diff
@@ -1,6 +1,133 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
+extern crate serde_json;
+
+use std::process::{Command};
+use serde_json::Value;
+
+fn code(tx_to: &str, tx_from: &str, blk_no: &str, address: &str) {
+    let prefix = "{\"jsonrpc\":\"2.0\",\"method\":\"eth_getCode\",\"params\":[\"";
```

Comment:
Can we use `serde_json` to build the JSON string? This looks like a hack and things like escaping strings might break.